.github/workflows/test.yml

```
@@ -13,7 +16,7 @@ jobs:
       - name: Node.js setup
         uses: actions/setup-node@v1
```

Feel free to use `actions/setup-node@v2`.

```suggestion
        uses: actions/setup-node@v2
```

---

I don't mind upgrading but I'm considering that to be out of scope for this Pull Request. E.g., when changing this I would also consider changing `runs-on: ubuntu-18.04` to something more recent. But I'd like to avoid that discussion in this thread. Feel free to open a Pull Request for this though :slightly_smiling_face: 

To clarify, I consider the other change int this file in scope because I want some certainty that everything actually works on these version of Node.

---

np. :+1:

---

I'm planning to address anything CI related in https://github.com/birjolaxew/svglint/pull/48, feel free to chime in with more suggestions if you want @LitoMore :slightly_smiling_face: